CHANGELOG.md

```diff
@@ -20,4 +20,15 @@
 - 🪲 Fix the bug occurring with weekly, monthly and annual time series which failed to convert to offsets
   
 
+## Version 1.0.0 - Initial release - 2019-11
```

Comment:
Maybe add the visual recipes as subsections to version 1.0.0, for consistency with the 2.0.0 section?

Reply:
done! latest version : https://github.com/dataiku/dss-plugin-timeseries-preparation/blob/chore/v200-add-v1-changelog/CHANGELOG.md